test/e2e/cli/pkg_test.go

```diff
@@ -75,7 +75,7 @@ var _ = Describe("kraft pkg", func() {
 					Expect(err).ToNot(HaveOccurred())
 
 					Expect(stderr.String()).To(BeEmpty())
-					Expect(stdout).To(MatchRegexp(`^Retrieve new lists of Unikraft components, libraries and packages.\n`))
+					Expect(stdout.String()).To(MatchRegexp(`^Retrieve new lists of Unikraft components, libraries and packages.\n`))
```

Comment:
Can you swash a0f8c35c8e61ac622ef22ef2a5af5b16fd594615 and 6c3f8e53f79e02478cf45e294a25732408332ebd together and adjust the commit message -- these two changes work hand-in-hand.